src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -60,7 +60,7 @@ public void run() {
     };
 
     @Inject
-    ClientAnalytics(
+    public ClientAnalytics(
```

Comment:
My main complaint is that not making the constructor public  forces others to create their own builders in the `com.arcbees.analytics.client` package to be able to instantiate a noop client or the regular one.